.github/workflows/self-smoke-test-action.yml

```diff
@@ -3,8 +3,10 @@
 name: 🧪
 
 on:   # yamllint disable-line rule:truthy
-  push:
   pull_request:
+  workflow_run:
```

Comment:
Two separate workflow runs is often hard to track. Instead, I adopted a practice of modularizing the workflow pieces as reusable workflows having the `reusable-` prefix in their names. This allows embedding everything in all the right places. Let's try this, WDYT?

Reply:
I will return to this suggestion at a later time.

Reply:
I think we should be able to circle back to this already...

Reply:
> Two separate workflow runs is often hard to track. Instead, I adopted a practice of modularizing the workflow pieces as reusable workflows having the `reusable-` prefix in their names. This allows embedding everything in all the right places. Let's try this, WDYT?

@webknjaz I removed `workflow_call` and switched to a reusable workflow instead.